src/wallet/wallet.h

```diff
@@ -291,6 +291,11 @@ struct CRecipient
     CTxDestination dest;
     CAmount nAmount;
     bool fSubtractFeeFromAmount;
+
+    friend bool operator==(const CRecipient& a, const CRecipient& b)
+    {
+        return a.dest == b.dest && a.nAmount == b.nAmount;
```

Comment:
In 78d1426fabd7b5b6cdd28de508a1692fce291174 "Add BIP352 test vectors as unit tests"

Shouldn't this also check sffo too?

```suggestion
        return a.dest == b.dest && a.nAmount == b.nAmount && a.fSubtractFeeFromAmount == b.SubtractFeeFromAmount;
```

Reply:
Maybe? If we want to know if the actual objects themselves are equal, then yes, but if we wan't a more conceptual "are these the same transaction output," then I'd argue no?